core/vm/contracts_lightclient_test.go

```diff
@@ -102,4 +104,34 @@ func TestTmHeaderValidateAndMerkleProofValidate(t *testing.T) {
+	moranValidator := iavlMerkleProofValidateMoran{}
+	res, err := moranValidator.Run(bytest)
+	assert.Errorf(t, err, "invalid merkle proof")
```

Comment:
can we compare the error message here

Reply:
`assert.Errorf ` is actually comparing the error message, unfortunately, we can only get high-level error message `invalid merkle proof`.